src/main.cc

```diff
@@ -2,26 +2,30 @@
 #include <memory>
 
 #include "gmsh.h"
+#include "io.h"
 
 int main(int argc, char** argv) {
   try {
     if (argc != 3) {
```

Comment:
we no longer need the dimension of the problem in `main`. Remove dimension argument.